### PR TITLE
Onboarding Refactoring

### LINKED ISSUE #54: Refactor onboarding
Currently we have the OnboardingFragment which is not part of the design anymore. Also there are some minor issues with the rest of the screens (sizes, titles, etc)

Current design v1.7: https://www.figma.com/file/Gjdy62Pi2wctPPhdenqF36/Covid-Watch---App-v1.7?node-id=0%3A1

### PR DESCRIPTION
**Pull request by BurningAXE**
- Deleted screen not present in new design
- Updated layouts to better reflect new design and fit on screen
- Renamed TitleScreen to SplashScreen so there is a match between design and implementation

### CODE REVIEW

**Review by nkumarcc — APPROVED** [review]
Lgtm! In the future for test plans put the UI, but I'm down with this.

File: app/src/main/java/org/covidwatch/android/ui/SplashFragment.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package org.covidwatch.android
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.navigation.findNavController
+import org.covidwatch.android.databinding.FragmentSplashBinding
```
Comment (nkumarcc) [inline app/src/main/java/org/covidwatch/android/ui/SplashFragment.kt:10]: I know in the other PR @Apisov mentioned not putting template code in the Fragment. I'm okay with for now, but what are the alternatives?

**Comment by nkumarcc** [comment]
One other thing; sizes are still pretty spread outside of the `dimens.xml`. Personally okay with that while doing MVP but we should do them where we can

**Comment by BurningAXE** [comment]
Yeah, there's definitely a lot of refactoring ahead if we stick with this UI and not Madi's.

File: app/src/main/java/org/covidwatch/android/ui/SplashFragment.kt (hunk 1 above)
Comment (BurningAXE) [reply app/src/main/java/org/covidwatch/android/ui/SplashFragment.kt:10]: What template code do you refer to - I think I removed everything.

Comment (nkumarcc) [reply app/src/main/java/org/covidwatch/android/ui/SplashFragment.kt:10]: Setting the `onClickListener`. Also to be clear idk if there's an alternative/this falls in the exact same category

Comment (BurningAXE) [reply app/src/main/java/org/covidwatch/android/ui/SplashFragment.kt:10]: That's required wiring for the button to lead to the next screen. 
What Pavlo was refering to were the unused `Intent` parameters that come with a new Fragment template and were left by whoever initially created it.

**Pull request merged**